index.html

```diff
@@ -13,10 +13,70 @@
       type="text/css"
     />
     <link rel="stylesheet" href="css/style.css" />
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
     <link rel="shortcut icon" type="image/x-icon" href="favicon.ico" />
+    <title>My Vlog</title>
   </head>
   <body>
-    <!-- Add your HTML markup here -->
-    <!-- Remember: Use semantic HTML tags like <header>, <main>, <nav>, <footer>, <section> etc -->
+    <div class="header">
+
+<div class="topnav">
+  <a class="active1" href="#home">Home</a>
+  <a  class="active2" href="#contact">Contact</a>
+  <a class="active3"  href="#about">About me</a>
+</div>
+      <h2>Iran</h2>
+    </div>
+    
+    <div class="row">
+      <div class="leftcolumn">
+        <div class="card">
+          <h2>About Iran:</h2>
+          <h5>Iran, a mountainous, arid, and ethnically diverse country of southwestern Asia. Much of Iran consists of a central desert plateau, which is ringed on all sides by lofty mountain ranges that afford access to the interior through high passes. Most of the population lives on the edges of this forbidding, waterless waste. The capital is Tehrān, a sprawling, jumbled metropolis at the southern foot of the Elburz Mountains. Famed for its handsome architecture and verdant gardens, the city fell somewhat into disrepair in the decades following the Iranian Revolution of 1978–79, though efforts were later mounted to preserve historic buildings and expand the city’s network of parks. As with Tehrān, cities such as Eṣfahān and Shīrāz combine modern buildings with important landmarks from the past and serve as major centres of education, culture, and commerce.</h5>
```

Comment:
This should probably be a paragraph. Also, if you have a long paragraph, you can divide it in multiple sentences in the html file. The browser will ignore these new lines, so feel free to break into short sentences. This makes it easier to review also.